commands/build.go

```diff
@@ -321,12 +322,15 @@ func runBuild(ctx context.Context, dockerCli command.Cli, options buildOptions)
 	if err != nil {
 		return err
 	}
+
 	_, err = b.LoadNodes(ctx)
 	if err != nil {
 		return err
 	}
 	driverType := b.Driver
 
+	dockerUsingContainerdSnapshotter := isDockerUsingContainerdSnapshotter(ctx, dockerCli, opts.Exports)
```

Comment:
This is doing (multiple) extra requests, so should be avoided if it is unnecessary. Could be avoided completely I think when connected with "load".

Reply:
Thanks; can you expand a bit more on what "connected with `load`" means?